src/pages/jaesangChoi/Main/Feeds/Feeds.js

```diff
@@ -73,19 +121,27 @@ class Feeds extends Component {
 
           <div className="third_comment">42분 전</div>
 
+          <ul className="comment_list">
+            {comments.map(t => {
```

Comment:
t라는 변수명을 가지고는 여기에 할당된 값이 뭐일지 전혀 유추가 안되는 것 같습니다!
comments 배열안의 하나하나의 요소들이니까 comment라고 이름지어주는게 좋지 않을까요?
```suggestion
            {comments.map(comment => {
```